django_mysql/locks.py

```diff
@@ -144,9 +150,9 @@ def lock(self):
                 locks.append("{} WRITE".format(qn(name)))
             cursor.execute("LOCK TABLES {}".format(", ".join(locks)))
 
-    def unlock(self, exc_type, exc_value, traceback):
+    def unlock(self, *exc):
```

Comment:
This won't work if one doesn't pass in `exc_type, exc_value, traceback`, please refactor to:

`def unlock(self, exc_type=None, exc_value=None, traceback=None):`

and leave the `_atomic.__exit__` call unchanged